### DISCUSSION TITLE
v3.2.0

### DISCUSSION THREAD

**Discussion by rfelber** [secureCodeBox/secureCodeBox#738] [Releases]
## Changes
This release contains the following changes 🎉. 
Help spread the word or leave a GitHub star if you like it 😉
  
![GitHub commits since tagged version](https://img.shields.io/github/commits-since/secureCodeBox/secureCodeBox/v3.1.1) ![GitHub Repo stars](https://img.shields.io/github/stars/secureCodeBox/secureCodeBox?style=social) ![Twitter URL](https://img.shields.io/twitter/url?style=social&url=https%3A%2F%2Fgithub.com%2FsecureCodeBox%2FsecureCodeBox%2Freleases%2Ftag%2Flatest)

## 🚀 Features

- Added Option to Automatically Retrigger ScheduledScan on ScanType Changes @J12934 (#672)
- Added `ftp-anon` and `ftp-banner` nmap scripts and support for portrule scripts @EndPositive (#692)
- Added ImagePullPolicy to all scanner and parser images (HelmCharts) @rseedorff (#657)
- Added Github Actions to track new scanner version releases  @Ilyesbdlala (#710)
- Fixed a Multitude of Makefile and integration test issues and style fixes @EndPositive (#688)

## 🚓 Security Scanner

- Add presigned head url to scan object @EndPositive (#690)
- Added a new Joomla Scanner `cmseek` @Ilyesbdlala (#664)
- Fixed scantype for `nuclei`scanner @fuhrmeistery (#700)
- Updated `AMASS` scanner to latest release v3.14 @rseedorff (#694)
- Added `ftp-anon` and `ftp-banner` nmap scripts and support for portrule scripts @EndPositive (#692)
- Updated `gitleaks` scanner to latest release v.7.6.1 @rseedorff (#666)
- Updated `nuclei` to latest release v2.5.2 @rseedorff (#669)
- Updated `kubeaudit` to latest release v0.14.2 @rseedorff (#667)
- Updated `trivy` to latest release 0.19.2 @rseedorff (#670)
- Updated `WPScan` scanner to latest release v3.8.19 @rseedorff (#665)
- Implemented init container support for scanners (ADR-0009) @malexmave (#724)

## 🐛 Bug Fixes

- Braces around the links causes docusaurus build fail @Weltraumschaf (#736)
- Helm Chart: Juice-Shop - Ingress installation failed because of 'apiVersion not set' @fbuchmeier-abi (#698)
- Fixed scantype for `nuclei`scanner @fuhrmeistery (#700)
- Added defaults for cascading inherit fields and disable omitEmpty @EndPositive (#687)
- Fixed installation scripts @SebieF (#651)
- Fixed change indent for .mk files to tab @fuhrmeistery (#678)
- Fixed Defectdojo Hook Issue Causing It to Not Run on Openshift Clusters @J12934 (#673)

## 🧪 Test

- Fixed the failing Typo3Scan Integration tests @Ilyesbdlala (#721)
- Added a Joomla demo-target @Ilyesbdlala (#681)
- Decrease Number of Expected Trivy Findings within the integration-test @J12934 (#671)
- Adding integration tests for trivy security scanner @SebieF (#650)

## 📚 Documentation

- Added ADR-0009 for container pre-populating specs @malexmave (#711)
- Updated ADR-0009 with decision @malexmave (#718)
- Sign the DCO @malexmave (#708)

## 🔧 Maintenance

- :construction: [Consistency] Making all `scan-type.yaml` files consistent @SebieF (#703)
- :construction: [Consistency] Making all `values.yaml` files consistent @SebieF (#714)
- :construction: [Consistency] Make all files of `cmseek` follow consistent templates @SebieF (#732)
- :construction: [Consistency] Create New Scanner makefile command @SebieF (#706)
- :construction: [Consistency] Making all `Chart.yaml files` consistent @SebieF (#702)
- :construction: [Consistency] Making all `cascading-rules.yaml` files consistent @SebieF (#704)
- :construction: [Consistency] Added empty file handling and test to all parsers @SebieF (#705)
- Updated npm deps ./bin/npm-audit-fix-all.sh @fuhrmeistery (#697)
- Added Targets To Build README and Docs @fuhrmeistery (#677)
- Cache Operator and Lurker Images In CI @fuhrmeistery (#662)
- Cache SDK Images As Artifacts @fuhrmeistery (#655)
- Align The Makefile For auto-discovery @fuhrmeistery (#675)
- Updated the Docker Meta Action @fuhrmeistery (#660)
- Added Pipeline For Release Image Builds @fuhrmeistery (#653)
- Added a minio startup helper script @SebieF (#652)
- Added missing modules to new/reworked GH workflows @rseedorff (#737)
- Build Scanners With New Make Targets @fuhrmeistery (#712)

## 📌 Dependencies

- [Snyk] Upgrade ajv from 8.6.2 to 8.6.3 @snyk-bot (#696)
- Upgrade `AMASS` scanner to latest release v3.14 @rseedorff (#694)
- Updated `gitleaks` scanner to latest release v.7.6.1 @rseedorff (#666)
- Updated `nuclei` to latest release v2.5.2 @rseedorff (#669)
- Updated `kubeaudit` to latest release v0.14.2 @rseedorff (#667)
- Updated `trivy` to latest release 0.19.2 @rseedorff (#670)
- Updated `WPScan` scanner to latest release v3.8.19 @rseedorff (#665)

## Distribution
[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/secureCodeBox)](https://artifacthub.io/packages/search?repo=securecodebox)
[![Docker Hub](https://img.shields.io/badge/Docker%20Hub-secureCodeBox-blue?style=flat&logo=docker&color=lightblue&labelColor=blue&logoColor=white)](https://hub.docker.com/u/securecodebox) 

## Contributers
Thx to all our contributers supporting this project 🤗
@EndPositive, @Ilyesbdlala, @J12934, @SebieF, @Weltraumschaf, @fbuchmeier-abi, @fuhrmeistery, @malexmave, @rseedorff and @snyk-bot

<hr /><em>This discussion was created from the release <a href='https://github.com/secureCodeBox/secureCodeBox/releases/tag/v3.2.0'>v3.2.0</a>.</em>

**Comment by J12934** [comment]
Some of the scanners helm chart weren't published correctly due to a bug in the helm chart publishing pipeline.
We're looking into it and should hopefully have all charts correctly updated in the next couple of hours 🤞

**Reply by J12934** [reply]
Should be resolved now, if you think another chart is missing with v3.2.0 please let us know 🙏
